core/server/config/utils.js

```diff
@@ -69,6 +70,21 @@ exports.getContentPath = function getContentPath(type) {
 };
 
 /**
+* Check if the URL in config has a protocol and sanitise it if not including a warning that it should be changed
+*/
+exports.sanitiseUrlProtocol = function sanitiseUrlProtocol() {
+    var url = this.get('url');
+
+    if (url.match(/^https?:\/\//i)) {
+        return;
+    } else {
+        this.set('url', 'http://' + url);
+        console.log(chalk.red('Watch out! Your URL in the config should have a protocol! E.g. "http://my-ghost-blog.com"'));
+        console.log(chalk.white('Current URL: ') + chalk.cyan(url));
```

Comment:
We have a couple of start messages in `core/ghost-server`.
I would like to keep them in one place. There is already a url stdout for development mode.